android/app/src/main/res/layout/activity_parent_quest_details.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
+            android:id="@+id/quest_image_view"
+            android:src="@drawable/broom"
```

Comment:
I'll remove all this hard-coded data once requests to server are made for quests